src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -188,6 +188,10 @@ var DOMPropertyOperations = {
     }
     if (value == null) {
       node.removeAttribute(name);
+    } else if (value === true) {
+      node.setAttribute(name, '');
+    } else if (value === false) {
+      node.removeAttribute(name);
```

Comment:
Also, using "true" and "false" but not making it a boolean attribute seems like a really confusing decision 🤔